hypothesis/lms
# Store the source of course copy assignments  as `copied_from`

Store from which assignment the current one was copied from, only for new assignment. 


### Testing:


- Reset the DB
```tox -qe dockercompose -- exec postgres psql -U postgres -c "truncate grouping,assignment cascade;"; make devdata;```

- Launch https://aunltd.brightspacedemo.com/d2l/le/content/6861/viewContent/2524/View as a teacher

- Check the data in the DB:


```tox -qe dockercompose -- exec postgres psql -U postgres -c "select id, resource_link_id, copied_from_id from assignment where resource_link_id = 'F7373A6B-A469-426E-8F99-138CEF897E95-1200_6782' or copied_from_id is not null;"```
